client/src/pages/private/ParticipantTable.js

```diff
@@ -175,36 +144,6 @@ export default () => {
     }));
   };
```

Comment:
would it make sense to bring these functions into the reducer as well? I kind of want everything that uses setOrder or setPagination to sit in the same place, could that work effectively?

Reply:
I'm all for it!